### ISSUE TITLE
Navbar Component

### ISSUE THREAD

**Issue by Radilx** [Drem-Labs/drem-interface#64]
Implement the Navbar component. This component will appear at the top of every screen, and will need to be complete before we can finalize any of the pages on Interface 3 ([Figma](https://www.figma.com/file/VKVQRAFTJIfIw6vc6kD18I/Drem-MVP?node-id=3410%3A19517)). The Navbar component should have:

- Drem Logo
- Search Bar - #39 
- Menu Component - #15 

NOTE: This issue contains has multiple dependencies. Please sure that all three of the above dependencies are resolved (and merged) before tackling this issue.

DOD: Make a navbar component that is responsive for mobile/web using the above components. 

![Group 37519](https://user-images.githubusercontent.com/22382740/194771588-fc42e1c4-8809-4ce9-81cd-57b43137d6ed.png)

**Comment by lpolt** [comment]
This issue utilizes components that already exist in the repo. this is also a test for the webhook

**Comment by Radilx** [comment]
Yeah, it looks like an intern tackled the menu component a few months ago -- it was just never merged. I'll see if it can be merged now.
